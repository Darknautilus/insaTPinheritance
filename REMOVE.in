R rectangle1 56 46 108 4536 
L l172v 56 46 108 4536 
C circle1 12 15 45
OA agg1 rectangle1 circle1
Oa agg2 l172v agg1
LIST
DELETE rectangle1 
LIST 
DELETE agg1 
LIST
DELETE circle2
LIST
