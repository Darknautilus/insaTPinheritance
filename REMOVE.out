OK
#New object: rectangle1
OK
#New object: l172v
OK
#New object: circle1
OK
#New object: agg1
OK
#New object: agg2
rectangle1
l172v
circle1
agg1
agg2
OK
#Delete object: rectangle1
l172v
circle1
agg1
agg2
OK
#Delete object: agg1
l172v
circle1
agg2
ERR
#Invalid parameters
l172v
circle1
agg2
