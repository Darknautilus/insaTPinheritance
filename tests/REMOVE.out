OK
OK
OK
OK
OK
rectangle1 56 46 108 4536 
l172v 56 46 108 4536 
circle1 12 15 45
agg1 rectangle1 circle1
agg2 l172v agg1
OK
l172v 56 46 108 4536 
circle1 12 15 45
agg1 rectangle1 circle1
agg2 l172v agg1
OK
l172v 56 46 108 4536 
circle1 12 15 45
agg2 l172v agg1
ERR
#Invalid parameters
l172v 56 46 108 4536 
circle1 12 15 45
agg2 l172v agg1
